README.md

```diff
@@ -319,6 +319,16 @@ Disable or enable support for device network-time requests (LoRaWAN MAC request
 
 If disabled, stub routines are provided that will return failure (so you don't need conditional compiles in client code).
 
+### Battery level report
+
+To support battery level indication in the MAC `DevStatusAns` messages you can either use an `int LMIC_registerBattLevelCb(lmic_battlevel_cb_t *pBattLevelCb, void *pUserData)` call to register a callback function of the form `typedef u1_t lmic_battlevel_cb_t(void *pUserData);`, if you have `LMIC_ENABLE_user_events` enabled, or call `LMIC_setBattLevel(u1_t battLevel)` periodically.
```

Comment:
> if you have `LMIC_ENABLE_user_events` enabled

The default in lmic is to support user events. So the above text maybe better should read "if you don not have disabaled ..."

Reply:
Quite possibly. I am using this library in a custom-built platform specific setup, so I am not that aware of the standard defaults :)